backend/game/migrations/0001_initial.py

```diff
@@ -59,7 +61,8 @@ class Migration(migrations.Migration):
                 ('monster_name', models.CharField(max_length=30)),
                 ('username', models.CharField(max_length=30)),
                 ('password', models.CharField(max_length=30)),
```

Comment:
remove passwords?

Reply:
True removed previously

Reply:
Migration was not done but redid the migration to reference updated model my mistake